_playwright-tests/tests/API/Snapshots.spec.ts

```diff
@@ -160,4 +163,62 @@ test.describe('Snapshots', () => {
       }
     });
   });
+
+  test('test_list_snapshot_errata', async ({ client, cleanup }) => {
+    const repoName = `snapshot-erratra-${randomName()}`;
+    const repoUrl = 'https://content-services.github.io/fixtures/yum/comps-modules/v1/';
+
+    await cleanup.runAndAdd(() => cleanupRepositories(client, repoName, repoUrl));
+
+    let repo: ApiRepositoryResponse;
+    await test.step('Create repo with 6 errata', async () => {
+      repo = await new RepositoriesApi(client).createRepository({
+        apiRepositoryRequest: {
+          name: repoName,
+          url: `https://stephenw.fedorapeople.org/fakerepos/multiple_errata/`,
+          snapshot: true,
+        },
+      });
+      expect(repo.name).toBe(repoName);
+      expect(repo.url).toBe(repoUrl);
```

Comment:
Hi, this is repoUrl from line 169, not same as line 178

Reply:
I already noticed it and fixed it locally, 
just hadn’t pushed the changes until now. 
Thanks for pointing it out!